server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -25,6 +25,13 @@ import qualified Data.Text                 as T
 import qualified Network.HTTP.Types        as HTTP
 import qualified Network.Wai.Extended      as Wai
 
+-- import qualified Data.ByteString.Lazy                   as BL
```

Comment:
We can remove these?

Reply:
Don't see why not. We can always reimport them if they're required.